### ISSUE TITLE
'distrobox enter wolfi' fails at "Installing basic packages..." stage.

### ISSUE THREAD

**Issue by hirnidrin** [ublue-os/bluefin#560]
### Steps

`just distrobox-wolfi` -> runs ok.
`distrobox enter wolfi` ->
```
Container wolfi is not running.
Starting container wolfi
run this command to follow along:
 podman logs -f wolfi
 Starting container...                  	 [ OK ]
 Installing basic packages...           	 Error: An error occurred
```

`podman logs -f wolfi` -> slightly shortened output
```
distrobox: Installing basic packages...
+ '[' -n  ]
+ printf 'distrobox: Installing basic packages...\n'
+ basename /bin/bash
+ shell_pkg=bash
+ '[' bash '=' ash ]
+ '[' 0 -ne 0 ]
+ command -v find
/usr/bin/find
+ command -v mount
/sbin/apk
+ command -v apk
+ '[' 0 -ne 0 ]
+ apk add bash
OK: 79 MiB in 72 packages
+ apk search -q mesa-dri
+ apk search -q mesa-vulkan
+ deps='
			bash
			bc
			curl
			diffutils
			findmnt
			findutils
			gnupg
			less
			lsof
			mount
			umount
			ncurses
			pinentry
			posix-libc-utils
			procps
			shadow
			su-exec
			sudo
			util-linux
			util-linux-misc
			vte3
			wget
			
			
			vulkan-loader
		'
+ install_pkg=
...
+ apk info procps
+ install_pkg=' bash curl diffutils findmnt findutils gnupg mount umount ncurses posix-libc-utils procps'
...
+ install_pkg=' bash curl diffutils findmnt findutils gnupg mount umount ncurses posix-libc-utils procps su-exec util-linux util-linux-misc wget vulkan-loader'
+ apk add bash curl diffutils findmnt findutils gnupg mount umount ncurses posix-libc-utils procps su-exec util-linux util-linux-misc wget vulkan-loader
(1/6) Installing gnupg (2.2.41-r0)
(2/6) Installing mount (2.39.2-r0)
(3/6) Installing libproc-2-0 (4.0.4-r0)
(4/6) Installing procps (4.0.4-r0)
ERROR: procps-4.0.4-r0: trying to overwrite usr/bin/kill owned by coreutils-9.4-r1.
ERROR: procps-4.0.4-r0: trying to overwrite usr/bin/uptime owned by coreutils-9.4-r1.
(5/6) Installing su-exec (0.2-r2)
(6/6) Installing vulkan-loader (1.3.264-r0)
1 error; 81 MiB in 78 packages
+ '[' 1 -ne 0 ]
Error: An error occurred
+ printf 'Error: An error occurred\n'
```

### Observations

* Containerfile.wolfi explicitly removes procps: https://github.com/ublue-os/bluefin/blob/d80bc8160075e329fb5694e46864515b24ec5f56/toolboxes/Containerfile.wolfi#L16
* distrobox-init then tries to re-add procps, which seems to cause the eror: https://github.com/89luca89/distrobox/blob/9309e1ebe5c70fd1f9b5286fd3a76f3e7bd995bd/distrobox-init#L448

**Comment by castrojo** [comment]
Yeah we need to spend more time on the wolfi box in general, it's just been a matter of not having enough time. Since it's broken anyway if you wanna PR fixes into it that'd be awesome!

**Comment by hirnidrin** [comment]
Thanks for the fix!

I digged into this also over the weekend. Findings:
* distrobox has improved a lot since the last official release 1.5.0.2 in June (which the rpm-ostree package is based on, I guess). There are about 200 new commits in the main branch (aka the 'next' version).
* The "Warning: there was a problem setting up the user" is caused by the missing 'useradd' bin in Wolfi. Latest distrobox versions work around this, especially note this commit https://github.com/89luca89/distrobox/commit/0e637ca00b6dd572f6a668ef77fbe9c6800fc24f 
* Latest distrobox versions themselves add all the packages that are right now in Containerfile.wolfi -> no need to add anything, distrobox enter / init will take care of it.

Here is a compare of a much simplified Containerfile.wolfi
https://github.com/hirnidrin/bluefin/commit/a60f505992ce76f5eedb3ffab8b5634e15e57fb7
But it works only with the latest distrobox version... which we can easily install with `just distrobox-git`. I have tested it, works fine, no errors on distrobox create / enter / init.

I don't know if you want to commit this to bluefin, as it depends on a non-rpm-ostree version of distrobox. We could also wait for distrobox official release 1.6 (or whatever it will be) before cleaning up wolfi-toolbox. Of course, I gladly submit a PR right now if you like :-)
